chatterbot_corpus/corpus.py

```diff
@@ -88,9 +88,10 @@ def load_corpus(self, dotted_path):
         for file_path in data_file_paths:
             corpus = self.read_corpus(file_path)
 
-            corpora.categories = corpus.get('categories', [])
-            conversations = corpus.get('conversations', [])
+            categories = corpus.get('categories', [])
+            corpora.categories.extend([categories])
```

Comment:
I'm a little bit confused about these changes. The corpora.categories is now a list of categories?